src/patternfly/components/Label/label.scss

```diff
@@ -208,6 +209,7 @@
   // Content
   --#{$label}__content--MaxWidth: 100%;
   --#{$label}__content--Gap: var(--pf-t--global--spacer--xs);
+  --#{$label}__content--Cursor: auto;
```

Comment:
Noticing an issue caused by this for the "add label", since the pf-v5-c-label__content class is on a div in that case instead of the button element:


https://github.com/patternfly/patternfly/assets/70952936/22dcb0b3-23ac-4b39-998c-2ba12e9a7f60